src/apps/ipv4/reassemble.lua

```diff
@@ -1,15 +1,14 @@
 module(..., package.seeall)
 
-local bit = require("bit")
-local constants = require("apps.lwaftr.constants")
-local ctable = require('lib.ctable')
-local ctablew = require('apps.lwaftr.ctable_wrapper')
-local ffi = require('ffi')
-local lwutil = require("apps.lwaftr.lwutil")
-local C = ffi.C
```

Comment:
Indeed, though it's often faster to not call C functions; and in this case we were able to remove all calls to ffi.C things.

To be honest I am of two minds.  One is, of course, anything we can do for perf is good :)  The other is that we have a lot of nasty code written in the name of perf, and maybe not all of that is needed even with the current LuaJIT constraints.  In the future we may be able to use FFI metatable early binding as in https://github.com/snabbco/snabb/issues/734 to make namespace caching not necessary; that sure would be nice!